hazelcast/src/main/java/com/hazelcast/client/impl/protocol/template/ResponseTemplate.java

```diff
@@ -172,4 +173,18 @@ void Exception(int errorCode, String className, @Nullable String message, StackT
     @Response(ResponseMessageConst.ENTRIES_WITH_CURSOR)
     void EntriesWithCursor(int tableIndex, List<Map.Entry<Data, Data>> entries);
 
+    @Since("1.4")
+    @Response(ResponseMessageConst.SCHEDULED_TASK_HANDLER)
+    void ScheduledTaskHandler(ScheduledTaskHandler handler);
+
+    @Since("1.4")
+    @Response(ResponseMessageConst.SCHEDULED_TASK_STATISTICS)
+    void ScheduledTaskStatistics(long createdAt, long firstRunStartNanos, long lastIdleTimeNanos,
+                                 long lastRunEndNanos, long lastRunStartNanos, long totalIdleTimeNanos,
+                                 long totalRuns, long totalRunTimeNanos);
+
+    @Since("1.4")
+    @Response(ResponseMessageConst.LIST_SCHEDULED_TASK_HANDLER)
```

Comment:
Do you really need such a type? You can just use Lis<ScheduledTaskHandler> as the response and not introduce a new response type.